docs/test-api-matrix.md

```diff
@@ -7,7 +7,7 @@ application|:white_check_mark: partially|:heavy_check_mark:||
 bucket||||partially within application
 dependency|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
 file|:heavy_check_mark:|:heavy_check_mark:||
-import||||
+import||:heavy_check_mark:|:heavy_check_mark:|
```

Comment:
A small update, please remove also "deeper test" (the second check mark), we'd need add also "negative" test covering rejected records etc. for a deeper test (but that don't have to be in this PR).